### DISCUSSION TITLE
Métamorph

### DISCUSSION THREAD

**Discussion by kekeishere** [Crownicles/Crownicles#1795] [Suggestions fermées]
On peux mettre un nouveau pet 5 ⭐ qui est personnalisable en gros ce sera un métamorphe  et on peux le paramétrer on pourra changer l'emoji et on pourra ajouter le type de la transformation du pet

Familier de KéKé² :
Type : 👹 Kaiju 
Nom scientifique : 🫠 Métamorphius
Surnom : Crimson Typhoon

Rareté : ⭐⭐⭐⭐ ⭐ 
Sexe : Mâle ♂️
Mentalité : 🙀 Craintif

un autre exemple

Familier de KéKé² :
Type : 📻 Radio BlaBla
Nom scientifique : 🫠 Métamorphius
Surnom : Rad-y-eau

Rareté : ⭐⭐⭐⭐ ⭐ 
Sexe : Mâle ♂️
Mentalité : 😻 Dressé

**Comment by Jaarzz** [comment]
L'idée me plaît, mais j'y trouve un problème. 
Vu qu'on pourrait changer l'émoji à sa guise, le pet se retrouverait à n'être plus un pet. Comme dans ton exemple, tu prends une radio, ce n'est pas un pet et cela perdrait tout son sens.

Aussi, le jeu doit être un maximum réaliste, ce qui ne serait plus le cas ici. 
Voilà ce que j'en pense

**Reply by kekeishere** [reply]
Je suis d'accord mais actuellement je n'ai pas trouvé d'idée pour empêcher sa c'est dur pour le reste

**Comment by BastLast** [comment]
Hello, trop compliqué techniquement par rapport aux avantages en gameplay. Désolé
